src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
-public class MainApp {
+import java.util.ArrayList;
 
+public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        ArrayList<Machine> arrayList = new ArrayList<Machine>();
+        arrayList.add(truck);
+        arrayList.add(bulldozer);
+        arrayList.add(excavator);
```

Comment:
```suggestion
        Machine[] machines = {truck, bulldozer, excavator};
```
let's do it one line